### PR TITLE
Add support for Deno v0.39.0

### LINKED ISSUE #7: Cannot install Deno v0.39.0
Hello there. Thanks for making this plugin!

I noticed that this plugin fails to install Deno v0.39.0.

```
$ asdf install deno 0.39.0
∗ Downloading and installing deno...

gzip: /home/uki00a/.asdf/installs/deno/0.39.0/bin/deno.gz: not in gzip format
Fail: Could not uncompress
Fail: An error occurred
```

I think this is probably due to the change of denoland/deno#4545.

Thanks.

### PR DESCRIPTION
**Pull request by uki00a**
This PR tries to resolve #7.

My bash skill is very poor, so any feedback is appreciated :smile:

### CODE REVIEW

File: bin/install [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 set -euo pipefail
 
+semver_lte() {
```
Comment (imbsky) [inline bin/install:5]: This function is not necessary and can be simpler by stripping out the dots and comparing them as integers.

**Comment by imbsky** [comment]
Also, just needs a update prerequirements section in README.md.

**Comment by imbsky** [comment]
This looks good overall. Let's make some minor changes and then merge 🙂

File: README.md [hunk 2]
```diff
@@ -14,7 +14,8 @@
 - Make sure you have the required dependencies installed:
   - curl
   - git
-  - gunzip
+  - gunzip (for installing v0.35.0 or lower)
+  - unzip (for installing v0.36.0 or higher)
```
Comment (imbsky) [inline README.md:18]:
```suggestion
  - gunzip (for v0.35.0 or lower)
  - unzip (for v0.36.0 or higher)
```

**Comment by imbsky** [comment]
Looks good to me.

**Comment by imbsky** [comment]
Thanks @uki00a for your contribution!

**Comment by imbsky** [comment]
@all-contributors please add @uki00a for code doc

**Comment by uki00a** [comment]
@imbsky Thanks for the review!

**Comment by imbsky** [comment]
Anytime 🙂

**Pull request merged**